README.md

```diff
@@ -10,6 +10,11 @@
 #### Panel
    * `.config/plasma-org.kde.plasma.desktop-appletsrc`
 
+# KDE Applications
```

Comment:
this should be at the end. 

As the name of this repo says, it is related to KDE configs, so they should be at the top. Krusader is separate program, it should not be first thing others will see when they find this repository